MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -243,16 +243,15 @@ public ServerConfiguration()
         public bool AllowClientLogUpload { get; set; } = true;
 
         /// <summary>
-        /// Gets or sets the dummy chapters duration in seconds.
+        /// Gets or sets a value indicating whether dummy chapter generation is enabled.
         /// </summary>
-        /// <value>The dummy chapters duration.</value>
-        public int DummyChapterDuration { get; set; } = 300;
+        public bool EnableDummyChapterGeneration { get; set; } = false;
```

Comment:
I still think it's better to use 0 or -1 to disable it. We need fewer options, not more